### DISCUSSION TITLE
[@types/jquery] assumes too much about $ and jQuery global variables

### DISCUSSION THREAD

**Discussion by josemigallas** [DefinitelyTyped/DefinitelyTyped#68853] [Issues with a @types package]
Regarding `@types/jquery@3.5.29`

I'm workin on a very big and old project with multiple versions of jQuery. Some of the old ones (v1 and v2) are handled by rails and injected automagically in the global scope. 

However, in the more recent features that are implemented with Wevpack+Typescript we use import jQuery v3 from node_modules. We `import $ from 'jquery'` and use it locally in some file, we do not export this to the global scope. However, typescript assumes that it is available everywhere and it is causing issues.

The culprit must be these lines https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5dddaaa384aaba8ba23840fb8132f2f9f800c9ef/types/jquery/misc.d.ts#L7336-L7337

IMHO the types should not assume jQuery is in the global scope by default. It should be the user who declares these vars manually in their `globals.d.ts` provided they export it:
```ts
import $ from 'jquery'

window.$ = $
```
In our project is messing up all files where we call `$` from the global scope expecting it to be v1 but webpack and the editor are throwing type errors assuming it's v3.

Here's an issue I opened, believing it was a TS error, with some additional context: https://github.com/microsoft/TypeScript/issues/57555

[a comment by an automated account is not shown]

**Comment by ekr3peeK** [comment]
I have the same issue. Because of these two lines, typescript is not complaining if I am not importing directly jQuery in a file that is using it and I do not want jQuery to clutter the global namespace. In the old version, our webpack was set up so that it supplied the global jQuery variable, but right now we are trying to change that, and it is really really hard to hunt down all the files, where jQuery was used globally.
